outputs.tf

```
@@ -1,19 +1,19 @@
 output "id" {
   description = "The ID of the Security Group"
-  value       = join("", aws_security_group.this.*.id)
+  value       = join("", aws_security_group.this.id)
```

sorry, meant literally the whole expression is just `aws_security_group.this.id`...
```suggestion
  value       = aws_security_group.this.id
```